examples/restyresolver/api/pets.py

```diff
@@ -0,0 +1,44 @@
+from connexion import NoContent
+from flask import Response
+import time
+from rfc3339 import rfc3339
+import json
+
+pets = {}
+
+
+def post(pet):
+    count = len(pets)
+    pet['id'] = count + 1
+    pet['registered'] = rfc3339(time.time())
+    pets[pet['id']] = pet
+    return Response(json.dumps(pet), status=201, mimetype='application/json')
```

Comment:
any reason you are manually doing JSON serialization?

I would expect to just use `return pet`